### DISCUSSION TITLE
Ipopt 3.14: Does It Contain Option for Mumps?

### DISCUSSION THREAD

**Discussion by anilvrao** [coin-or/Ipopt#724] [Q&A]
I know this is may be a stupid question, but I am having problems with compiling a Mex interface for IPOPT on an Apple Silicon processor using the linear solver Mumps.  My understanding is that Mumps is the default linear solver for Ipopt.  For some reason, when I complete the compilation and test the Mex file on the MATLAB examples, I receive the error that Mumps is not a valid option for "linear_solver":

*** Error using Ipopt Matlab interface: ***

Invalid value for IPOPT option "linear_solver"
*******************************************

I have compiled Mumps with Ipopt and also compiled the Mex file successfully, so I am really not sure what is happening.  I am using gcc, g++, and gfortran in my compilation.  

Thanks!

**Comment by svigerske** [comment]
Duplicate of #723.
